guide/intro/chapter-1/index.html

```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: What is Pygame
+next_page: [/guide/intro/chapter-2, Opening A Window]
+---
+
+<body>
+  <p>In this article, we'll go over what pygame is and how to install it.</p>
+
+  <h2>Table of contents</h2>
+  <p></p>
+  <ol id="toc">
+    <!--
+            To be filled by javascript
+            -->
+  </ol>
+
+  <h2><a id="introduction" class="js">Introduction</a></h2>
+  <p>
+    It's time to get started on your pygame journey, but what is pygame? Well
+    pygame is "a free and open-source cross-platform library for the development
+    of multimedia applications like video games using Python". As the
+    description suggests, it's mostly used to make video games - which is why in
+    this book we'll be learning techniques relevant to making games
+  </p>
+
+  <p>
+    Pygame provides the very basic utility to draw and manipulate pixels on to
```

Comment:
Grammar feels off here, have a suggestion

```suggestion
    Pygame aims to be a user-friendly library to draw and manipulate pixels onto
```